2018-06-28 19:00:39,701 - INFO - [:15]- **************************Mason(2018-06-28 19-00-39):This is Flask*******************
2018-06-28 19:00:39,701 - INFO - [:36]- Project func...waiting for image filename
2018-06-28 19:00:39,706 - INFO - [:88]-  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2018-06-28 19:01:11,048 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:01:11] "[33mGET / HTTP/1.1[0m" 404 -
2018-06-28 19:01:12,603 - INFO - [:23]- Call model begin...
2018-06-28 19:01:12,603 - INFO - [:26]- **************************Mason(2018-06-28 19-01-12):model*******************
2018-06-28 19:01:12,603 - INFO - [:137]- path=None
2018-06-28 19:01:12,603 - INFO - [:255]- model process error...Back to Flask
Traceback (most recent call last):
  File "/Users/mason/Mason/Git/project-streetlamp_detection/Link_W&G/wutong_version/Link_GW.py", line 153, in main
    filename=os.path.basename(filePath)#ÊèêÂèñÊñá‰ª∂Âêç
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/posixpath.py", line 114, in basename
    i = p.rfind('/') + 1
AttributeError: 'NoneType' object has no attribute 'rfind'
2018-06-28 19:01:12,608 - ERROR - [:1761]- Exception on /process/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1957, in make_response
    'The view function did not return a valid response. The'
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
2018-06-28 19:01:12,610 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:01:12] "[1m[35mGET /process/?Begin=1 HTTP/1.1[0m" 500 -
2018-06-28 19:01:19,509 - INFO - [:23]- Call model begin...
2018-06-28 19:01:19,509 - INFO - [:26]- **************************Mason(2018-06-28 19-01-19):model*******************
2018-06-28 19:01:19,509 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:01:19,509 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:01:19,725 - INFO - [:182]- HSV process...
2018-06-28 19:01:19,954 - INFO - [:188]- Segment...
2018-06-28 19:01:20,018 - INFO - [:196]- Segment process...
2018-06-28 19:01:20,543 - INFO - [:213]- Contours...
2018-06-28 19:01:20,579 - INFO - [:244]- Total time consuption (60 pisce):1.0700 second
2018-06-28 19:01:20,579 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:01:20,610 - ERROR - [:1761]- Exception on /process/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1982, in make_response
    reraise(TypeError, new_error, sys.exc_info()[2])
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1974, in make_response
    rv = self.response_class.force_type(rv, request.environ)
  File "/usr/local/lib/python2.7/site-packages/werkzeug/wrappers.py", line 921, in force_type
    response = BaseResponse(*_run_wsgi_app(response, environ))
  File "/usr/local/lib/python2.7/site-packages/werkzeug/test.py", line 923, in run_wsgi_app
    app_rv = app(environ, start_response)
TypeError: 'int' object is not callable
The view function did not return a valid response. The return type must be a string, tuple, Response instance, or WSGI callable, but it was a int.
2018-06-28 19:01:20,613 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:01:20] "[1m[35mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 500 -
2018-06-28 19:05:59,588 - INFO - [:15]- **************************Mason(2018-06-28 19-05-59):This is Flask*******************
2018-06-28 19:05:59,588 - INFO - [:36]- Project func...waiting for image filename
2018-06-28 19:05:59,593 - INFO - [:88]-  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2018-06-28 19:06:05,547 - INFO - [:23]- Call model begin...
2018-06-28 19:06:05,547 - INFO - [:26]- **************************Mason(2018-06-28 19-06-05):model*******************
2018-06-28 19:06:05,547 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:06:05,547 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:06:05,741 - INFO - [:182]- HSV process...
2018-06-28 19:06:05,973 - INFO - [:188]- Segment...
2018-06-28 19:06:06,037 - INFO - [:196]- Segment process...
2018-06-28 19:06:06,521 - INFO - [:213]- Contours...
2018-06-28 19:06:06,554 - INFO - [:244]- Total time consuption (60 pisce):1.0064 second
2018-06-28 19:06:06,554 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:06:06,597 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:06:06] "[37mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 200 -
2018-06-28 19:07:57,187 - INFO - [:23]- Call model begin...
2018-06-28 19:07:57,187 - INFO - [:26]- **************************Mason(2018-06-28 19-07-57):model*******************
2018-06-28 19:07:57,187 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:07:57,187 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:07:57,352 - INFO - [:182]- HSV process...
2018-06-28 19:07:57,557 - INFO - [:188]- Segment...
2018-06-28 19:07:57,612 - INFO - [:196]- Segment process...
2018-06-28 19:07:58,093 - INFO - [:213]- Contours...
2018-06-28 19:07:58,118 - INFO - [:244]- Total time consuption (60 pisce):0.9312 second
2018-06-28 19:07:58,118 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:07:58,148 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:07:58] "[37mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 200 -
2018-06-28 19:28:54,378 - INFO - [:15]- **************************Mason(2018-06-28 19-28-54):This is Flask*******************
2018-06-28 19:28:54,378 - INFO - [:36]- Project func...waiting for image filename
2018-06-28 19:28:54,383 - INFO - [:88]-  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2018-06-28 19:28:58,070 - INFO - [:23]- Call model begin...
2018-06-28 19:28:58,071 - INFO - [:26]- **************************Mason(2018-06-28 19-28-58):model*******************
2018-06-28 19:28:58,071 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:28:58,071 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:28:58,258 - INFO - [:182]- HSV process...
2018-06-28 19:28:58,483 - INFO - [:188]- Segment...
2018-06-28 19:28:58,542 - INFO - [:196]- Segment process...
2018-06-28 19:28:59,016 - INFO - [:213]- Contours...
2018-06-28 19:28:59,041 - INFO - [:244]- Total time consuption (60 pisce):0.9700 second
2018-06-28 19:28:59,041 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:28:59,072 - INFO - [:28]- Call model done, begin to return data.
2018-06-28 19:28:59,073 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:28:59] "[37mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 200 -
2018-06-28 19:30:34,428 - INFO - [:16]- **************************Mason(2018-06-28 19-30-34):This is Flask*******************
2018-06-28 19:30:34,428 - INFO - [:39]- Project func...waiting for image filename
2018-06-28 19:30:34,430 - INFO - [:88]-  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2018-06-28 19:30:39,021 - INFO - [:24]- Call model begin...
2018-06-28 19:30:39,021 - INFO - [:26]- **************************Mason(2018-06-28 19-30-39):model*******************
2018-06-28 19:30:39,021 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:30:39,021 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:30:39,205 - INFO - [:182]- HSV process...
2018-06-28 19:30:39,435 - INFO - [:188]- Segment...
2018-06-28 19:30:39,496 - INFO - [:196]- Segment process...
2018-06-28 19:30:39,980 - INFO - [:213]- Contours...
2018-06-28 19:30:40,004 - INFO - [:244]- Total time consuption (60 pisce):0.9827 second
2018-06-28 19:30:40,004 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:30:40,036 - INFO - [:29]- Call model done, begin to return data as type json.
2018-06-28 19:30:40,036 - ERROR - [:1761]- Exception on /process/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "app.py", line 31, in Baogan_project
    j= json.dumps(dictionary)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 244, in dumps
    return _default_encoder.encode(obj)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 207, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 270, in iterencode
    return _iterencode(o, 0)
  File "/usr/local/Cellar/python@2/2.7.15/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: array([[2358,  704],
       [2358,  365],
       [2537,  365],
       [2537,  704]]) is not JSON serializable
2018-06-28 19:30:40,043 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:30:40] "[1m[35mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 500 -
2018-06-28 19:31:41,061 - INFO - [:16]- **************************Mason(2018-06-28 19-31-41):This is Flask*******************
2018-06-28 19:31:41,061 - INFO - [:39]- Project func...waiting for image filename
2018-06-28 19:31:41,063 - INFO - [:88]-  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2018-06-28 19:31:44,262 - INFO - [:24]- Call model begin...
2018-06-28 19:31:44,262 - INFO - [:26]- **************************Mason(2018-06-28 19-31-44):model*******************
2018-06-28 19:31:44,263 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:31:44,263 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:31:44,450 - INFO - [:182]- HSV process...
2018-06-28 19:31:44,685 - INFO - [:188]- Segment...
2018-06-28 19:31:44,747 - INFO - [:196]- Segment process...
2018-06-28 19:31:45,249 - INFO - [:213]- Contours...
2018-06-28 19:31:45,278 - INFO - [:244]- Total time consuption (60 pisce):1.0158 second
2018-06-28 19:31:45,278 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:31:45,310 - INFO - [:29]- Call model done, begin to return data dict.
2018-06-28 19:31:45,318 - ERROR - [:1761]- Exception on /process/ [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1816, in full_dispatch_request
    return self.finalize_request(rv)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1831, in finalize_request
    response = self.make_response(rv)
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1982, in make_response
    reraise(TypeError, new_error, sys.exc_info()[2])
  File "/usr/local/lib/python2.7/site-packages/flask/app.py", line 1974, in make_response
    rv = self.response_class.force_type(rv, request.environ)
  File "/usr/local/lib/python2.7/site-packages/werkzeug/wrappers.py", line 921, in force_type
    response = BaseResponse(*_run_wsgi_app(response, environ))
  File "/usr/local/lib/python2.7/site-packages/werkzeug/wrappers.py", line 59, in _run_wsgi_app
    return _run_wsgi_app(*args)
  File "/usr/local/lib/python2.7/site-packages/werkzeug/test.py", line 923, in run_wsgi_app
    app_rv = app(environ, start_response)
TypeError: 'dict' object is not callable
The view function did not return a valid response. The return type must be a string, tuple, Response instance, or WSGI callable, but it was a dict.
2018-06-28 19:31:45,320 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:31:45] "[1m[35mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 500 -
2018-06-28 19:32:45,301 - INFO - [:16]- **************************Mason(2018-06-28 19-32-45):This is Flask*******************
2018-06-28 19:32:45,301 - INFO - [:39]- Project func...waiting for image filename
2018-06-28 19:32:45,304 - INFO - [:88]-  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2018-06-28 19:32:47,747 - INFO - [:24]- Call model begin...
2018-06-28 19:32:47,747 - INFO - [:26]- **************************Mason(2018-06-28 19-32-47):model*******************
2018-06-28 19:32:47,747 - INFO - [:137]- path=./images_feed/DJI_0016.jpg
2018-06-28 19:32:47,747 - INFO - [:158]- %%%%%%%%%%%%processing image:DJI_0016.jpg
2018-06-28 19:32:47,932 - INFO - [:182]- HSV process...
2018-06-28 19:32:48,164 - INFO - [:188]- Segment...
2018-06-28 19:32:48,226 - INFO - [:196]- Segment process...
2018-06-28 19:32:48,735 - INFO - [:213]- Contours...
2018-06-28 19:32:48,760 - INFO - [:244]- Total time consuption (60 pisce):1.0131 second
2018-06-28 19:32:48,760 - INFO - [:248]- process completed...back to Flask App
2018-06-28 19:32:48,792 - INFO - [:29]- Call model done, begin to return data dict.
2018-06-28 19:32:48,793 - INFO - [:88]- 127.0.0.1 - - [28/Jun/2018 19:32:48] "[37mGET /process/?path=./images_feed/DJI_0016.jpg HTTP/1.1[0m" 200 -
